datacenter/dtr/2.4/guides/support.md

```diff
@@ -5,12 +5,15 @@ description: Your Docker subscription gives you access to prioritized support. Y
 keywords: support, help
 ---
 
-Your Docker Data Center, or Docker Trusted Registry subscription gives you
-access to prioritized support. The service levels depend on your subscription.
+Your Docker Enterprise Edition subscription gives you access to prioritized
+support. The service levels depend on your subscription.
 
-If you need help, you can file a ticket via:
+Before reaching out to Docker Support, make sure you're listed as an authorized
+support contact for your account. If you're not listed as an authorized
+support contact, find a person that is, and ask them to open a case with
```

Comment:
s/that/who

People always get "who" and things always get "that".

Can you give them a quick link to the place where they can see if they are an authorized contact?

Reply:
I don't think we have something like that. From a user standpoint is not great that users have to learn how to navigate their internal orgs to get to the right person.
But from a security standpoint it's actually makes sense no to provide such tools since an attacker could gain information that otherwise they would not have access to.